src/AppInstallerCommonCore/Manifest/ManifestCommon.cpp

```diff
@@ -282,6 +282,74 @@ namespace AppInstaller::Manifest
         }
     }
 
+    ExpectedReturnCodeEnum ConvertToExpectedReturnCodeEnum(const std::string& in)
+    {
+        ExpectedReturnCodeEnum result = ExpectedReturnCodeEnum::Unknown;
+
+        if (Utility::CaseInsensitiveEquals(in, "PackageInUse"))

```

Comment:
>Utility::CaseInsensitiveEquals [](http://example.com/codeflow?start=12&length=30)

nit: since there're more than a couple strings, we might want to do toLower than do direct compare, like what we did for InstallerTypeEnum